.github/workflows/workflow.yaml

```diff
@@ -301,3 +304,19 @@ jobs:
                       create_output(result["Target"], misconfiguration)
 
               sys.exit(exit_status)
+
+  trivy-sbom:
+    name: Generate SBOM report of application in ${{ inputs.working-directory }}
```

Comment:
This could get a little shorter name. E.g.

> Generate SBOM from ${{ inputs.working-directory }}

Reply:
Fixed!